src/tools/pub/cmd/pub-add.md

```diff
@@ -192,6 +212,6 @@ in immediate dependencies (`--precompile`).
 To prevent precompilation, use `--no-precompile`.
 
 {{site.alert.info}}
-  *Problems?*
```

Comment:
And again

Reply:
> And again

Other than these indentations is everything okay?